DARPA-ASKEM/terarium
# Ts/3630 interventions for calibrate

# Description
**PLEASE ALSO REVIEW:** https://github.com/DARPA-ASKEM/pyciemss-service/pull/88

Here we are adding the interventions from the model configuration to our calibrate and optimize requests within the HMIServer. 

This will mean when a user connects a model config with interventions to a calibrate node and hits run the HMISERVER will add the model config's interventions to the request without the front end needing to worry about it

# Testing:

1. User should be able to add interventions in a model config
2. Run a simulation - and still see interventions impacting parameters
3. Run a calibration - and now see interventions impacting parameters
4. Run an optimization - and now see interventions impacting parameters
5. Saving a model config from optimize will carry over previous interventions as well as include any new ones from the optimize itself.


<img width="875" alt="image" src="https://github.com/DARPA-ASKEM/terarium/assets/17088680/3e0db3fd-ae76-494f-b1c0-da51ae3505fc">
